test/csuite/random_abort/main.c

```diff
@@ -198,6 +207,9 @@ thread_run(void *arg)
         }
         cursor->set_value(cursor, &data);
         testutil_check(cursor->insert(cursor));
+
+        testutil_check(session->commit_transaction(session, NULL));
```

Comment:
You might want to add a FIXME comment here too so that we remember to remove this line also.